addons/website/views/snippets/s_masonry_block.xml

```diff
@@ -243,17 +225,17 @@
     <div class="row o_grid_mode" data-row-count="10">
         <div class="o_grid_item col-lg-3 g-col-lg-3 g-height-10 text-center o_cc o_cc2" data-name="Block"
```

Comment:
Same vertical issue, the "centered" button in the WB is not selected.
Add `justify-content-center`

Same for the other blocks of this template ;)